example/offload-template/CMakeLists.txt

```diff
@@ -64,6 +64,6 @@ target_include_directories(run-offload PRIVATE
 # Use special linking macros to support CUDA+VecGeom when used
 celeritas_target_link_libraries(run-offload
   ${Geant4_LIBRARIES}
-  Celeritas::celeritas
-  Celeritas::accel
+  Celeritas::celeritas # for user action
+  Celeritas::G4
```

Comment:
```suggestion
  Celeritas::celeritas # for user action
  Celeritas::accel
  Celeritas::G4
```
here and in geant4.

Reply:
Nope , G4 should alias accel

Reply:
OK fixed it, thanks for the tip about aliasing in the config file!